sda-commons-server-auth-testing/src/test/java/org/sdase/commons/server/auth/testing/KeyLoaderProxyIT.java

```diff
@@ -21,41 +21,37 @@
 import org.apache.http.HttpHeaders;
 import org.junit.BeforeClass;
 import org.junit.ClassRule;
-import org.junit.Rule;
 import org.junit.Test;
 import org.junit.rules.RuleChain;
 import org.sdase.commons.server.auth.testing.test.AuthTestApp;
 import org.sdase.commons.server.auth.testing.test.AuthTestConfig;
 import org.sdase.commons.server.testing.EnvironmentRule;
-import org.sdase.commons.server.testing.LazyRule;
 import org.sdase.commons.server.testing.SystemPropertyRule;
 
 /** A test that checks if the jersey client that is used to load keys can use a proxy server */
 public class KeyLoaderProxyIT {
-  @ClassRule
-  public static final WireMockClassRule PROXY_WIRE =
+  private static final WireMockClassRule PROXY_WIRE =
       new WireMockClassRule(wireMockConfig().dynamicPort());
 
-  @ClassRule
-  public static final EnvironmentRule ENVIRONMENT_RULE =
+  private static final EnvironmentRule ENVIRONMENT_RULE =
       new EnvironmentRule().setEnv("AUTH_RULE", "{\"keys\": [{}]}");
```

Comment:
Can't we set this via config()?

Reply:
This is a tricky one. With today's knowledge, I guess we would have build the AuthRule in a different way. This `auth: ${AUTH_RULE}` entry in the config.yml is not very nice and can make the life hard. Better would have been to just set config overrides (`ConfigOverride.config("auth.keys", "whatever").addToSystemProperties()`) if possible (didn't tested it).

But I can try if it is really necessary here.

Reply:
Ah, right if we have an env variable there, we can't later reset it without having a custom config.yaml